cluster/gce/configure-vm.sh

```diff
@@ -428,6 +428,10 @@ function create-salt-kubelet-auth() {
     mkdir -p /srv/salt-overlay/salt/kubelet
     (umask 077;
       cat > "${kubelet_kubeconfig_file}" <<EOF
+{% set api_servers = "http://localhost:8080" -%}
```

Comment:
Why use jinja instead of just reading the environment variables (e.g. KUBELET_APISERVER) and setting the value directly?